app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/asset_type_for_non_uk_residents.scala.html

```diff
@@ -39,12 +40,24 @@
 returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link
 )
 
-@(form: Form[List[AssetType]], backLink: Call, hasCreatedDraftReturn: Boolean, isATrust: Boolean)(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+@(
+        form: Form[List[AssetType]],
+        backLink: Call,
+        hasCreatedDraftReturn: Boolean,
+        isATrust: Boolean,
+        representativeType: Option[Either[PersonalRepresentative.type, Capacitor.type]]
+)(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
 
 @key = @{"multipleDisposalsAssetTypeForNonUkResidents"}
 @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 @isAgent = @{ request.userType.contains(UserType.Agent) }
-@userKey = @{if (isAgent) ".agent" else if (isATrust) ".trust" else ""}
+@userKey = @{
+  if (representativeType.exists(_.isLeft)) ".personalRep"
+  else if (representativeType.exists(_.isRight)) ".capacitor"
```

Comment:
Any reason to change order here?
As we discussed, I thought of following this order:
```        if (repType.exists(_.isLeft)) ".personalRep"
        else if (repType.exists(_.isRight)) ".capacitor"
        else if (isAgent) ".agent"
        else if (isATrust) ".trust"
        else ""
```

Reply:
it was Andy's preference to keep them together, the only important bit is personalRep at the top - capacitor will not clash with agent or trust so it will work anywhere between top and bottom!